src/styles/App.scss

```diff
@@ -0,0 +1,10 @@
+@import '~@blueprintjs/core/lib/css/blueprint.css';
+@import '~@blueprintjs/icons/lib/css/blueprint-icons.css';
+@import '~@blueprintjs/core/lib/css/blueprint.css';
+@import '~@blueprintjs/icons/lib/css/blueprint-icons.css';
```

Comment:
It looks like all the blueprint imports are twice here (same as it was in App.css, dating back to our work session a few weeks ago -- I don't think this was introduced in this PR). We can probably remove the imports on line 3 and 4 at some point.